x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -191,6 +196,7 @@ export type AlertUpdates = Omit<Alert, 'id' | 'executionStatus'>;
 export interface AlertTableItem extends Alert {
   alertType: AlertType['name'];
   tagsText: string;
+  actionsCount: number;
```

Comment:
To work with the `EuiTableSortingType` interface, I had to add `actionsCount` here. I also renamed it from `actionsText` to avoid confusion because it contains a `number` value.